examples/tutorials/02_pipeline_operations.ipynb

```diff
@@ -15,7 +15,6 @@
    "source": [
```

Comment:
<p>Why not just use simple <code>"pipeline"</code>  name here and below?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/analysiscenter/batchflow/pull/332/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>